2020q1/riscv.md

```diff
@@ -7,6 +7,8 @@ Contact: Mitchell Horne <mhorne@FreeBSD.org>
 Contact: John Baldwin <jhb@FreeBSD.org>  
 Contact: Kristof Provost <kp@FreeBSD.org>  
 Contact: Philip Paeps <philip@FreeBSD.org>  
+Contact: [freebsd-riscv Mailing List](https://lists.FreeBSD.org/mailman/listinfo/freebsd-riscv)  
```

Comment:
Would this make more sense in the Link section above?

Reply:
I'm fine for both. I added it here because I feel it is a way to contact. Please change it if you want.